SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -290,12 +292,21 @@ FLocalDeploymentManager::ERuntimeStartResponse FLocalDeploymentManager::StartLoc
 	// runtime.exe --config=squid_config.json --snapshot=snapshots/default.snapshot --worker-port 8018 --http-port 5006 --grpc-port 7777
 	// --worker-external-host 127.0.0.1 --snapshots-directory=spatial/snapshots/<timestamp>
 	// --schema-bundle=spatial/build/assembly/schema/schema.sb
-	// --event - tracing - logs - directory = `<Project > / spatial / localdeployment / <timestamp> / `
+	// --event-tracing-logs-directory=`<Project>/spatial/localdeployment/<timestamp>/`
 	FString RuntimeArgs =
 		FString::Printf(TEXT("--config=\"%s\" --snapshot=\"%s\" --worker-port %hu --http-port=%hu --grpc-port=%s "
 							 "--snapshots-directory=\"%s\" --schema-bundle=\"%s\" --event-tracing-logs-directory=\"%s\" %s"),
-						*LaunchConfig, *SnapshotName,  WorkerPort,  HTTPPort, *FString::FromInt(RuntimeGRPCPort), *CurrentSnapshotPath, *SchemaBundle,
-						*EventTracingPath, *LaunchArgs);
+						*LaunchConfig, *SnapshotName, WorkerPort, HTTPPort, *FString::FromInt(RuntimeGRPCPort), *CurrentSnapshotPath,
+						*SchemaBundle, *EventTracingPath, *LaunchArgs);
+
+	if (bEnableSessionLogRecording)
+	{
+		FString SessionLogsDir = FPaths::Combine(LocalDeploymentLogsDir, TEXT("session_logs"));
+		PlatformFile.CreateDirectoryTree(*SessionLogsDir);
```

Comment:
Perhaps we should check that the path exists at after calling CreateDirectoryTree?